Makefile

```diff
@@ -1,7 +1,14 @@
 # The development version of clang is distributed as the 'clang' binary,
 # while stable/released versions have a version number attached.
 # Pin the default clang to a stable version.
+GOARCH := $(shell go env GOARCH)
+ifneq ($(GOARCH),arm64)
 CLANG ?= clang-12
+else
+# Ubuntu Focal used for the ARM64 builder container does not provide
+# clang-12 out of the box
+CLANG ?= clang-11
```

Comment:
Unfortunately we need clang-12 for upcoming CO-RE support, see #301. FWIW I don't think we need to build using arm64 clang, there should be no difference in output for bpfel / bpfeb targets on either platform.